src/actor/model.rs

```diff
@@ -417,6 +452,20 @@ where
                     }
                 ))
             }
+            ActorModelAction::Crash(id) => {
+                let index = usize::from(id);
+                match last_state.actor_states.get(index) {
+                    None => return None,
+                    Some(last_actor_state) => Some(format!(
+                        "{}",
+                        ActorStep {
+                            last_state: &**Cow::Borrowed(last_actor_state),
+                            next_state: None,
+                            out: Out::new() as Out<A>,
+                        }
+                    )),
+                }
```

Comment:
Can probably reduce some code w/ a `map` here. Not a blocker for merging of course.